app/controllers/admin/choices_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ChoicesController < ApplicationController
```

Comment:
There is no need to create a Choice controller because we only need WORD controller.
Please delete this file.